states/vermont/elections.toml

```diff
@@ -8,7 +8,55 @@ date = 2020-08-11
 name = "Congressional Primary"
 original_date = 2020-08-11
 
+[20200811.registration]
+in_person_by = 2020-08-11T19:00:00
+# online earlier (ensure name appears on checklist)
+# but can register up to election day
+online_by = 2020-08-07
+sources = ["https://sos.vermont.gov/elections/voters/registration/", "https://olvr.vermont.gov"]
+
+[20200811.poll]
+in_person_start = 2020-08-11
```

Comment:
I think this is 10am on election day based on the calendar.

Reply:
Adding poll close time of 7pm too